packages/datatrak-web/src/Routes.tsx

```diff
@@ -35,9 +36,17 @@ import { CentredLayout, BackgroundPageLayout, MainPageLayout } from './layout';
  * If the user is logged in and tries to access the login page, redirect to the home page
  */
 const LoggedInRedirect = ({ children }) => {
-  const { isLoggedIn, isLoading, isFetched } = useUser();
-  if (isLoading || !isFetched) return <FullPageLoader />;
-  if (isLoggedIn) return <Navigate to="/" replace={true} />;
+  const { isLoggedIn, isLoading, isFetched, data } = useUser();
+  if (isLoading || !isFetched) {
+    return <FullPageLoader />;
+  }
+  if (isLoggedIn) {
```

Comment:
If we wanted to tidy this up ever so slightly, we could do 
`if(!isLoggedIn) return children` and then we don't need to nest the if statements for logged in state

Reply:
yes much better thank you